test/tools/Makefile

```diff
@@ -6,7 +6,13 @@
 ARCH                    ?= $(subst x86_64,amd64,$(shell uname -m))
 BUILTIN_CLOUD_PROVIDERS ?= aws azure ibmcloud vsphere libvirt
 GOFLAGS                 ?=
-GOOPTIONS               ?= GOOS=linux GOARCH=$(ARCH) CGO_ENABLED=0
+GOOS 					?= $(shell uname)
```

Comment:
NP, but make sure you watch the spacing here and use spaces instead of tabs before the `?=` where you space out the variable and the value to be assigned to it.